.github/workflows/pages.yml

```diff
@@ -32,39 +31,22 @@ jobs:
           python-version: '3.10'
       - name: Setup Pages
         uses: actions/configure-pages@v2
-
       - name: Install Dependencies
         run: |
           python -m pip install --quiet --upgrade pip setuptools wheel 
           python -m pip install -r docs/docsgen/source/requirements.txt
-
       - name: Uninstall onnx
         run: python -m pip uninstall -y onnx
-
       - name: Install onnx development version
         run: |
           sudo apt-get install libprotobuf-dev protobuf-compiler
           git submodule update --init --recursive
           export CMAKE_ARGS="-DONNX_USE_PROTOBUF_SHARED_LIBS=ON"
           python setup.py install
-
       - name: Build Docs
-        # TODO: fix warnings from "make html -j auto" so it can normally succeed without errors
-        continue-on-error: true
         run: |
           cd docs/docsgen
-          make html -j auto > stdout_stderror.log 2>&1
-      - name: Check whether previous build docs succeeded
-        run: |
-          cat docs/docsgen/stdout_stderror.log
-          if grep -F "build succeeded" docs/docsgen/stdout_stderror.log
-          then
-            echo "Build succeeded."
-          else
-            echo "Build failed."
-            exit 1
-          fi
```

Comment:
More context about why manually check the error log: https://github.com/onnx/onnx/pull/4570#issuecomment-1268809300. To confirm: have those build errors resolved by this PR? If not, probably we still need to keep this code.

Reply:
It only fails when you use `make -j auto`, which I removed.

When I run the original workflow the docs were not generated but the build still passed.

Hence to make this workflow more reliable, I removed the hack and now the build takes a little longer (~5mins). I considered this a worthwhile tradeoff